doc/classes/Shader.xml

```diff
@@ -55,7 +55,7 @@
 		<member name="code" type="String" setter="set_code" getter="get_code" default="&quot;&quot;">
 			Returns the shader's code as the user has written it, not the full generated code used internally.
 		</member>
-		<member name="custom_defines" type="String" setter="set_custom_defines" getter="get_custom_defines" default="&quot;&quot;">
+		<member name="custom_defines" type="String" setter="set_custom_defines" getter="get_custom_defines" default="&quot;#define MAX_LIGHT_DATA_STRUCTS 409#define MAX_FORWARD_LIGHTS 8#define MAX_REFLECTION_DATA_STRUCTS 455#define MAX_SKELETON_BONES 1365&quot;">
```

Comment:
CC @m4gr3d - looks like the new `custom_defines` gets a non-null default value.

Reply:
I could only find reference to that default value in [rasterizer_scene_gles3.cpp](https://github.com/godotengine/godot/blob/909e303def2c16c00bfa9426f4d78f2f70b84f3a/drivers/gles3/rasterizer_scene_gles3.cpp#L5153). Is that expected?